ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -199,4 +218,34 @@ open class ContactView: UIView {
 
         subtitleLabel = label
     }
+
+    private func setupPressedStateOverlay() {
+        pressedStateOverlay.backgroundColor = Colors.Contact.pressedState
+        pressedStateOverlay.clipsToBounds = true
+        pressedStateOverlay.frame = avatarView.frame
+        pressedStateOverlay.isHidden = true
+        pressedStateOverlay.isUserInteractionEnabled = false
+        pressedStateOverlay.layer.cornerRadius = avatarView.frame.width / 2
+    }
+
+    @objc private func touchDownHandler() {
+        pressedStateOverlay.isHidden = false
+    }
+
+    @objc private func touchUpInsideHandler() {
+        contactViewDelegate?.didTapContactView?(self)
+        pressedStateOverlay.isHidden = true
+    }
+
+    @objc private func touchMovedHandler() {
+        pressedStateOverlay.isHidden = true
```

Comment:
I don't think we want to hide the overlay just because the user moved his finder? Typically, the dimmed state of a tappable control goes away when the touch event exits the bounds of the control's view.

Reply:
That is what I originally thought as well. After playing around with Outlook on iOS, it seems like the the dimmed state goes away when I tap the contact and drag.

Example: Tap a contact by the picture and then drag to the name of the contact. The dimmed state disappears. (The same behaviour occurs if tap and drag from the name to the picture of the same contact)